### DISCUSSION TITLE
Alert Rule; 'not-confirmed by LoTW'

### DISCUSSION THREAD

**Discussion by oh2gba** [foldynl/QLog#488] [Ideas]
Hello, 

I would like an Alert Rule filter that only displays spots marked as 'not confirmed by LoTW'.

Use case: DXCC award hunting

Regards,
John

**Comment by aa5sh** [comment]
Couldn't you just enable the memberlist for LoTW and have that enabled for the alert so it will only show those using LoTW.  Or are you wanting those that don't use LoTW?  
![image](https://github.com/user-attachments/assets/1fca2e41-df03-489a-ac71-821d9deb89a2)

**Comment by foldynl** [comment]
that is a very good point.

**Comment by oh2gba** [comment]
Thanks for your answer, 

I'm trying to set up an alert for countries I haven't confirmed on LoTW yet. 

Your suggestion would still send alerts for countries I already have confirmed, just because they’re on a new band I haven't worked on yet.

**Reply by aa5sh** [reply]
I think I understand better what you are wanting and I do see value to that.  Basically you would only want to consider it as being confirmed if it was using LoTW, that way if you have something confirmed only on Paper and have the opportunity to get it on LoTW you would like that.  I don't believe that there is a way to accomplish that today.  

The easiest way I could think to implement that would be to have an overall option on like a confirmed option where you could select LoTW, Paper or Both, then that could be taken into account in queries like the DxccStatus.  What are your thoughts Ladislav?  I do see value to that especially now that ARRL still isn't accepting paper cards for DXCC only LoTW.

**Reply by foldynl** [reply]
Opened an issue #508
